### PR TITLE
feat: delete annotations

### LINKED ISSUE #374: Annotations MVP: As a user I can delete an annotation
GIVEN I am looking at a dashboard cell 
AND there are annotations visible in my current time range

WHEN I hover over an annotation
THEN I see a tooltip containing time, summary, (optionally) message, a trash icon and an edit button

WHEN I mouse into the tooltip and click the trash icon
THEN I am prompted to confirm the action

WHEN I confirm the delete
THEN the Annotation is deleted
AND I see a notification fire (5s duration, success, "[AnnotationName] successfully deleted!"

UI is mostly built out for this story, just needs to be wired up and actually delete
AnnotationCard.tsx

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #374 

- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 23 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 89 | :green_circle:
pwa | 22 | :red_circle:

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 25 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 89 | :green_circle:
pwa | 22 | :red_circle:

**Review by hoorayimhelping — CHANGES_REQUESTED** [review]
great work, a couple of changes

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 42 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: src/annotations/actions/thunks.ts [hunk 1]
```diff
@@ -23,3 +35,16 @@ export const writeThenFetchAndSetAnnotations = (
...
   fetchAndSetAnnotations()(dispatch)
 }
+
+export const deleteAnnotations = annotation => async (
+  dispatch: Dispatch<AnnotationAction | PublishNotificationAction>
+) => {
+  try {
+    await deleteAnnotation(annotation)
+    dispatch(deleteAnnotationAction(annotation))
+    dispatch(notify(deleteAnnotationSuccess()))
```
[the remark this answers is not in the document]
Comment (zoesteinkamp) [reply src/annotations/actions/thunks.ts:45]: added!

**Pull request closed without merging**